systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -1031,16 +1031,23 @@ void testScaleConnectAndConnectorSubresource() {
     }
 
     @Test
+    @SuppressWarnings({"checkstyle:MethodLength"})
     void testMountingSecretAndConfigMapAsVolumesAndEnvVars() {
         String secretPassword = "password";
         String encodedPassword = Base64.getEncoder().encodeToString(secretPassword.getBytes());
 
         String secretEnv = "MY_CONNECTOR_SECRET";
         String configMapEnv = "MY_CONNECT_CONFIG_MAP";
 
+        String dotedSecretEnv = "MY_DOTED_CONNECTOR_SECRET";
+        String dotedConfigMapEnv = "MY_DOTED_CONNECT_CONFIG_MAP";
+
         String configMapVolumeName = "connect-config-map";
         String secretVolumeName = "connect-secret";
 
+        String dotedConfigMapVolumeName = "connect.config.map";
+        String dotedSecretVolumeName = "connect.secret";
+
```

Comment:
Maybe we should put this into some method? It's the same for ConnectST and ConnectS2IST. Same with creation of serets/cm

Reply:
Well the tests are practically same, only difference is between the CRs -> I would like to make the tests "more generic" how we discussed offline. But for now, I guess, I can create some method in `AbstractST` and then just refactor it.
WDYT @Frawless ?

Reply:
Discussed offline